.github/workflows/python-publish.yml

```diff
@@ -4,8 +4,9 @@
 name: Upload Python Package
 
 on:
-  release:
-    types: [created]
+  push:
```

Comment:
As i understand, with this syntax, the upload will only be triggered on push, i.e. not when i create a release online?

Reply:
An online release should push a new tag to the repository, although I haven’t tested this, but I assume that creating a release through GitHub - to the git repo - looks the same as pushing a tag. It’s possible that the action trigger somehow differentiates where the tag has been created, but I will test if it also works for “GitHub releases”.